contracts/samples/TokenPaymaster.sol

```diff
@@ -149,16 +149,13 @@ contract TokenPaymaster is BasePaymaster, UniswapHelper, OracleHelper {
     /// @dev This function is called after a user operation has been executed or reverted.
     /// @param context The context containing the token amount and user sender address.
     /// @param actualGasCost The actual gas cost of the transaction.
-    function _postOp(PostOpMode, bytes calldata context, uint256 actualGasCost) internal override {
+    function _postOp(PostOpMode, bytes calldata context, uint256 actualGasCost, uint gasPrice) internal override {
         unchecked {
             uint256 priceMarkup = tokenPaymasterConfig.priceMarkup;
             (
                 uint256 preCharge,
-                uint256 maxFeePerGas,
-                uint256 maxPriorityFeePerGas,
                 address userOpSender
-            ) = abi.decode(context, (uint256, uint256, uint256, address));
-            uint256 gasPrice = getGasPrice(maxFeePerGas, maxPriorityFeePerGas);
+            ) = abi.decode(context, (uint256, address));
```

Comment:
This means the entire `getGasPrice` function is no longer needed - please remove.